<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>1101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Adress">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Items">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Phone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>660</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>700</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>740</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>780</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="D12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>820</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>890</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>840</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Confirm">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>960</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background: white}
QPushButton {color: black}
QPushButton:hover{background:rgb(202, 202, 202)}</string>
   </property>
   <property name="text">
    <string>Confirm </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>960</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background: white}
QPushButton {color: black}
QPushButton:hover{background:rgb(202, 202, 202)}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>860</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QLabel {color: White}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color: White}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Homegrown Grill</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delivery </string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>391</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {background: rgb(85, 170, 0)}


QTextBrowser {opacity:0}
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>890</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {background: rgb(85, 170, 0)}


QTextBrowser {opacity:0}
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>890</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {background: rgb(85, 170, 0)}


QTextBrowser {opacity:0}
</string>
   </property>
  </widget>
  <zorder>textBrowser_3</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>textBrowser</zorder>
  <zorder>Info</zorder>
  <zorder>Adress</zorder>
  <zorder>Items</zorder>
  <zorder>Phone</zorder>
  <zorder>D1</zorder>
  <zorder>D3</zorder>
  <zorder>D4</zorder>
  <zorder>D2</zorder>
  <zorder>D8</zorder>
  <zorder>D9</zorder>
  <zorder>D10</zorder>
  <zorder>D7</zorder>
  <zorder>D6</zorder>
  <zorder>D5</zorder>
  <zorder>D11</zorder>
  <zorder>D12</zorder>
  <zorder>Box</zorder>
  <zorder>Total</zorder>
  <zorder>Confirm</zorder>
  <zorder>Back</zorder>
  <zorder>Total_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
